tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -223,17 +190,13 @@ private static void TryAddEndPoint(Span span)
             }
         }
 
-        private static void AddAppsecSpecificInstrumentations()
+        private static void AddAppsecSpecificInstrumentations(bool enable = true)
```

Comment:
nit: probably easier to understand if we rename this method and make the parameter mandatory

Reply:
Fixed in `73581a7a8289380ad851b8b4da85b66cbe880508`